setup.py

```diff
@@ -29,10 +29,31 @@
                  install_requires=[
                      'numpy',
                      'pybind11>=2.5.0',
-                     'colorama',
                      'setuptools',
                      'astor',
                  ],
+                 tests_requires=[  # To run test with Taichi:
```

Comment:
AFAIK, it's better to list packages that are required by the tests/test-runners in `tests_requires`, not the test runners themselves. Also, since we don't use `python setup.py` to install (I guess?), `tests_require` may not work as you'd expect here (it works when installing a package with `python setup.py test`). An alternative might be:

```
'extras_require': {
    'testing': ['pytest', 'pytest-rerunfailures', 'pytest-xdist', 'autograd'],
},
```
and then install as needed with `pip install taichi[testing]`